core/src/com/mygdx/game/models/Round.java

```diff
@@ -0,0 +1,114 @@
+package com.mygdx.game.models;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * The round keeps track of current players and their
+ * BrainstormingPhase, EliminationPhase, Brains being used the round,
+ * and which brain the player is currently on.
+ *
+ * players: List of current players
+ * brainstorminPhases: a Map that keeps track of which BrainstormingPhase belongs to which player
+ * eliminationPhases: a Map that keeps track of which EliminationPhase belongs to which player
+ * brains: a map of which Brains are being used this round and which player it belongs to.
+ * currentBrainNumbers: a map that keeps track of which brain each player is on
+ * maxSelectedBrains: the max nr of selected brains in the eliminationphase
+ * inEliminationPhase: A boolean to tell which round is going
+ *
+ * */
+
+public class Round {
+
+    private ArrayList<Player> players;
+    private HashMap<Player, BrainstormingPhase> brainstormingPhases = new HashMap<>();
+    private HashMap<Player, EliminationPhase> eliminationPhases = new HashMap<>();
+    private HashMap<Player, ArrayList<Brain>> playersBrains = new HashMap<>();
+    private HashMap<Player, Integer> currentBrainNumbers = new HashMap<>();
+    private int maxSelectedBrains;
+    private boolean inEliminationPhase = false;
+
+
+    /**
+     * A constructor that initializes the brainstormingphases for each player.
+     * @param players: List of current players
+     * @param brains: List of brains used this round, if round one these brains should contain no ideas
+     * @param maxHitPoints: The max HP for the wall
+     * @param BRAIN_DAMAGE: The amound of damage the brain does
```

Comment:
```suggestion
     * @param BRAIN_DAMAGE: The amount of damage the brain does
```
Minor grammar mistake